src/main/webapp/consentGroupDocuments/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ConsentGroupDocuments from './ConsentGroupDocuments';
+
+ ReactDOM.render(
+  <ConsentGroupDocuments
+    attachmentsUrl = {component.attachmentsUrl}
+    serverURL = {component.serverURL}
+    user = {component.user}
+    attachDocumentsUrl = {component.attachDocumentsUrl}
+    projectKey = {component.projectKey}
+    approveDocumentUrl = {component.approveDocumentUrl}
+    rejectDocumentUrl = {component.rejectDocumentUrl}
+    isAdmin = {component.isAdmin}
```

Comment:
See previous comments about this - I think this should be part of the `user` object and not necessarily a separate prop.